backend/models/notification.js

```diff
@@ -188,6 +195,34 @@ module.exports = {
 			});
 	},
 
+	/**
+	 * This function inserts a modelUpdateNotification for
+	 *
+	 * @param {*} teamSpace
+	 * @param {*} modelId
+	 * @param {*} revision
+	 * @returns {Promise< Array<username:string,notification:Notification> >} It contains the newly created notifications and usernames
+	 *
+	 */
+	insertModelUpdatedNotifications: async function(teamSpace, modelId, revision) {
+		const allUsers = await User.getAllUsersInTeamspace(teamSpace);
+		const usersAndPerms = await Promise.all(allUsers.map(async user => {
```

Comment:
It seems like all you do is extract the access part (line 214) of the object? should we just return access instead?